profiles/base.yaml

```diff
@@ -748,13 +748,15 @@ astral_plane_training:
   train_destination:
   train_source:
 predict_event: false  # Deprecated
-astrology_use_full_pools:
+astrology_use_full_pools: false
+astrology_pool_target: 1
```

Comment:
this should be astrology_pool_target: not astrology_pool_target: 1? having it 1 will always make it 1 with your other change.

Reply:
conditional assignment would also accomplish this:
```ruby
target = astrology_use_full_pools ? 10 : astrology_pool_target
```

Reply:
Updated Astrology.lic to match.